blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,82 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/2018-07-09-dvteam-activated
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories: student-posts

```

Comment:
Hi @artnerdnet ! Thanks for the PR. Please remove 'news' from the category and replace it with 'student-posts'.